test/e2e/components/vl-proza-message.js

```diff
@@ -1,9 +1,10 @@
 const { VlElement } = require('vl-ui-core').Test;
+const { VlButton } = require('vl-ui-button').Test;
 const { By, Key } = require('selenium-webdriver');
```

Comment:
Ik zou de By importeren vanuit vl-ui-core.Test.Setup.
De Key kan momenteel nog niet, spijtig genoeg, maar misschien moeten we dat ook doen in de toekomst.
Op die manier gaat er enkel een rechtstreekse dependency op selenium-webdriver in vl-ui-core zitten.